src/diffusers/models/unet_3d_blocks.py

```diff
@@ -74,6 +84,53 @@ def get_down_block(
             upcast_attention=upcast_attention,
             resnet_time_scale_shift=resnet_time_scale_shift,
         )
+    if down_block_type == "DownBlockMotion":
+        return DownBlockMotion(
+            num_layers=num_layers,
+            in_channels=in_channels,
+            out_channels=out_channels,
+            temb_channels=temb_channels,
+            add_downsample=add_downsample,
+            resnet_eps=resnet_eps,
+            resnet_act_fn=resnet_act_fn,
+            resnet_groups=resnet_groups,
+            downsample_padding=downsample_padding,
+            resnet_time_scale_shift=resnet_time_scale_shift,
+            temporal_norm_num_groups=temporal_norm_num_groups,
+            temporal_cross_attention_dim=temporal_cross_attention_dim,
+            temporal_num_attention_heads=temporal_num_attention_heads,
+            temporal_attention_bias=temporal_attention_bias,
+            temporal_activation_fn=temporal_activation_fn,
+            temporal_max_seq_length=temporal_max_seq_length,
```

Comment:
```suggestion
            num_layers=num_layers,
            in_channels=in_channels,
            out_channels=out_channels,
            temb_channels=temb_channels,
            add_downsample=add_downsample,
            resnet_eps=resnet_eps,
            resnet_act_fn=resnet_act_fn,
            resnet_groups=resnet_groups,
            downsample_padding=downsample_padding,
            resnet_time_scale_shift=resnet_time_scale_shift,
            temporal_norm_num_groups=temporal_norm_num_groups,
            temporal_cross_attention_dim=temporal_cross_attention_dim,
            temporal_num_attention_heads=temporal_num_attention_heads,
            temporal_attention_bias=temporal_attention_bias,
            temporal_activation_fn=temporal_activation_fn,
            temporal_max_seq_length=temporal_max_seq_length,
```
Do we really need to pass all these arguments? Let's maybe try to set more defaults that hold true for all motion modules. I'm a bit surprised that we need to allow for that much customization right at the beginning.